pkg/roachprod/roachprod.go

```diff
@@ -894,10 +895,10 @@ type StopOpts struct {
 // DefaultStopOpts returns StopOpts populated with the default values used by Stop.
 func DefaultStopOpts() StopOpts {
 	return StopOpts{
-		ProcessTag: "",
-		Sig:        9,
-		Wait:       false,
-		MaxWait:    0,
+		ProcessTag:  "",
+		Sig:         9,
```

Comment:
Super nit: If you opt to use the unix package, can probably refer to SIGKILL here as well.